src/pages/Governance/Card.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useContext } from 'react'
+import styled, { ThemeContext } from 'styled-components'
+import Card from '../../components/Card'
+import { AutoRow } from '../../components/Row'
+import { Text } from 'rebass'
+import CurrencyLogo from '../../components/CurrencyLogo'
+import DoubleCurrencyLogo from '../../components/DoubleLogo'
+import { transparentize } from 'polished'
+import { Currency } from 'dxswap-sdk'
+import { useRouter } from '../../hooks/useRouter'
+
+const LightCardWrap = styled(Card)`
+  border: 1px solid ${({ theme }) => transparentize(0.3, theme.bg2)};
+  background-color: ${({ theme }) => transparentize(0.3, theme.bg1)};
+  padding: 0.8rem;
+  width: calc(25% - 6px);
+  height: 96px;
+  display: flex;
+  flex-wrap: wrap;
+  cursor: pointer;
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    width: calc(33% - 4px);
+  `};
+  ${({ theme }) => theme.mediaWidth.upToExtraSmall`
+    width: calc(50% - 4px);
+  `};
+`
+
+const StyledText = styled(Text)``
+
+interface CardProps {
+  name: string
+  pairs: number
+  proposals?: number
+  currency: Currency
+}
+
+export const GovernanceCard = ({ name, pairs, proposals, currency }: CardProps) => {
+  const theme = useContext(ThemeContext)
+  const shittyCounter = 2
+  const router = useRouter()
+  const onClick = () => router.push(`/governance/${currency.symbol}/pairs`)
+
+  return (
+    <LightCardWrap onClick={onClick}>
```

Comment:
Would be great to make this generic, since it's useful for the farming stuff too. Maybe just a card with the double or single currency icon (depending if you pass in only one token or 2, to represent a pair) and title and a generic body that can be expressed as passed in `children`?

Reply:
https://github.com/levelkdev/dxswap-dapp/pull/170/commits/2441099428f996a484f76aa5da4923ad90c09e3d